examples/react-app/src/App.tsx

```diff
@@ -54,7 +75,9 @@ function App() {
           </div>
           <div className="section">
             <span>User</span>
-            <button onClick={() => user.setUserId('user-id')}>Set User Id</button>
+            <button onClick={() => user.setUserId('alpha-user-id')}>Set User Id (generic)</button>
+            <button onClick={() => user.setUserId('alpha-codegen-user-id')}>Set User Id (codegen)</button>
```

Comment:
TODO: Need to update experiment segments for this user id. https://experiment.amplitude.com/amplitude/289220/config/6173/configure